config/i18n-tasks.yml

```diff
@@ -107,8 +111,12 @@ search:
 # - '{devise,simple_form}.*'
 
 ## Consider these keys used:
-# ignore_unused:
-# - 'activerecord.attributes.*'
+ignore_unused:
+- 'activerecord.models.*'
+- 'activerecord.attributes.*'
```

Comment:
Why are these two lines needed? What attributes are we not translating?

Reply:
@joemasilotti  if you check the view part of it, I have removed the hardcoded text on labels (within form for each attribute) - for some reason they are not recognized by this gem but as per rails guides and best practices we should translate attributes on the active record and everywhere (incl. errors) they will apear the same.

It could be that I am missing some knowledge here on this gem.

Reply:
Gotcha, works for me. Thanks!